modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/common/RunResultUtility.java

```diff
@@ -20,85 +21,103 @@
 import dev.galasa.framework.spi.ResultArchiveStoreException;
 import dev.galasa.framework.spi.teststructure.TestMethod;
 import dev.galasa.framework.spi.teststructure.TestStructure;
+import dev.galasa.framework.api.common.Environment;
+import dev.galasa.framework.api.common.SystemEnvironment;
+import dev.galasa.framework.api.common.EnvironmentVariables;
 
 public class RunResultUtility {
-	
-	public static @NotNull RasRunResult toRunResult(@NotNull IRunResult runResult, boolean isMethodDetailsExcluded) throws ResultArchiveStoreException {
-	   
-	    String runId = runResult.getRunId();
+
+	Environment env;
+
+	public RunResultUtility() {
+		this.env = new SystemEnvironment();
```

Comment:
One last change please - can we update this line to `this(new SystemEnvironment());` please? It does the same thing by just calling the other constructor, but this is more consistent with several other classes where we do the same thing :)